src/components/MessageList/utils/getGroupStyles.js

```diff
@@ -0,0 +1,81 @@
+export const getGroupStyles = (messagesWithDates, noGroupByUser) => {
```

Comment:
how do we feel about passing through 1 param as an object for these instead of multiple params where we need to get the right order when calling it
```suggestion
export const getGroupStyles = ({ messagesWithDates, noGroupByUser }) => {
```